app_dart/test/service/luci_build_service_test.dart

```diff
@@ -0,0 +1,316 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:core';
+
+import 'package:cocoon_service/cocoon_service.dart';
+import 'package:cocoon_service/src/model/appengine/service_account_info.dart';
+import 'package:cocoon_service/src/model/luci/buildbucket.dart';
+import 'package:cocoon_service/src/request_handling/exceptions.dart';
+import 'package:cocoon_service/src/service/luci_build_service.dart';
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+
+import '../src/datastore/fake_cocoon_config.dart';
+
+void main() {
+  ServiceAccountInfo serviceAccountInfo;
+  FakeConfig config;
+  MockBuildBucketClient mockBuildBucketClient;
+  group('buildsForRepositoryAndPr', () {
+    setUp(() {
+      serviceAccountInfo = const ServiceAccountInfo(email: 'abc@abcd.com');
+      config = FakeConfig(deviceLabServiceAccountValue: serviceAccountInfo);
+      mockBuildBucketClient = MockBuildBucketClient();
+    });
+    test('emptyResponse', () async {
+      when(mockBuildBucketClient.batch(any)).thenAnswer((_) async {
+        return const BatchResponse(
+          responses: <Response>[
+            Response(
+              searchBuilds: SearchBuildsResponse(
+                builds: <Build>[],
+              ),
+            ),
+          ],
+        );
+      });
+      final LuciBuildService service =
+          LuciBuildService(config, mockBuildBucketClient, serviceAccountInfo);
+      final Map<String, Build> builders =
+          await service.buildsForRepositoryAndPr('cocoon', 1, 'abcd');
```

Comment:
The function is named "buildsFor.." therefore it's expected to returns "builds". However the receiving variable is named "builders".

Reply:
Fixed!